text/0000-default-scoped-light.md

```diff
@@ -0,0 +1,132 @@
+
+However, this is a footgun for developers. LWC developers are used to using `*.css` and having those styles "scoped" (per shadow DOM scoping semantics). These developers may forget to use `*.scoped.css` for light DOM components, in which case their component will "work," but unknowingly, their styles will leak outside of the component, potentially across the entire document.
+
+Changing the default behavior of `*.css` for light DOM components should nudge most developers towards the preferred solution – scoped styles. They will have to make an explicit decision to leak styles, by using `*.unscoped.css`.
+
+## Detailed design
+
+In the proposed system, developers can use any combination of the three filenames for the same component. E.g.:
+
+```
+component
+  ├──component.html
+  ├──component.css
+  ├──component.scoped.css
+  └──component.unscoped.css
+```
+
+These three filenames can be used with either shadow DOM or light DOM components.
+
+As described above, `*.scoped.css` and `*.unscoped.css` always behave the same (scoped and unscoped, respectively), whereas `*.css` will vary depending on whether the component is light (scoped) or shadow (unscoped).
+
+<details><summary>Implementation details</summary>
+
+Currently we use `*.scoped.css` as a signal for whether a stylesheet should be scoped or not, and the `@lwc/template-compiler` appends a [`?scoped=true` suffix](https://github.com/salesforce/lwc/blob/cfee90b9b2f03fe0242cd1366b394e0ee7373227/packages/%40lwc/compiler/src/transformers/template.ts#L86-L87) to the URL of the import to signal this. E.g.:
+
+```js
+import _implicitStylesheets from './component.css';
+import _implicitScopedStylesheets from './component.scoped.css?scoped=true';
+```
+
+In the new system, the `?scoped=true` suffix would vary depending on whether the importer is a light DOM or shadow DOM template. In light DOM, it would look like this:
+
+```js
+import _implicitStylesheets from './component.css?scoped=true';
+import _implicitScopedStylesheets from './component.scoped.css?scoped=true';
+import _implicitUnscopedStylesheets from './component.scoped.css';
+```
+
+...whereas in shadow DOM, it would be:
+
+```js
+import _implicitStylesheets from './component.css';
+import _implicitScopedStylesheets from './component.scoped.css?scoped=true';
+import _implicitUnscopedStylesheets from './component.scoped.css';
+```
+
+</details>
+
+## Drawbacks
+
+### Breaking existing behavior
+
+Light DOM is considered "beta" and has not officially shipped in LEX yet. However, some folks may be using it off-core. In those cases, the behavior will change, and they will have to fix their existing components, if they were relying on `*.css` being unscoped for light DOM components.
+
+### More complexity in the design
+
+The old system (`*.css` is unscoped, `*.scoped.css` is scoped) was simple to describe. The new system has a 3x3 table to explain how it works.

```

Comment:
Super duper nitpick: It's either a 2X3 or a 4X3 matrix if you include the labels. 🤣

```suggestion
The old system (`*.css` is unscoped, `*.scoped.css` is scoped) was simple to describe. The new system has a 2x3 table to explain how it works.

```